pkg/apis/serving/v1alpha2/configmap.go

```diff
@@ -53,6 +53,8 @@ type PredictorConfig struct {
 
 	DefaultImageVersion    string `json:"defaultImageVersion"`
 	DefaultGpuImageVersion string `json:"defaultGpuImageVersion"`
+	// Default timeout of predictor for serving a request, in milliseconds
+	DefaultTimeout string `json:"defaultTimeout,omitempty"`
```

Comment:
should be int64 instead of string

Reply:
@theofpa I am thinking we probably do not need this as TFServing already has a default 30s timeout right?

Reply:
We need to have it inherit the timeout from the inferenceservice though. When it comes to large timeout values, it's a problem.

Reply:
addressed in afaf6f9